eth/tracers/native/rip7560_validation.go

```diff
@@ -271,34 +298,44 @@ func (b *rip7560ValidationTracer) OnOpcode(pc uint64, op byte, gas, cost uint64,
 			if !rOk && !wOk {
 				access.Reads[slotHex] = b.env.StateDB.GetState(addr, slot).Hex()
 			}
-		} else {
+		} else if opcode == "STORE" {
```

Comment:
The opcode is called `SSTORE` not `STORE` so I assume this "if" will never be entered. Is it covered by a test?

Reply:
Yes, fixed and added test cases specific to SSTORE